examples/sphere/shallow_water.jl

```diff
@@ -18,25 +19,23 @@ import OrdinaryDiffEq
 using OrdinaryDiffEq: ODEProblem, solve, SSPRK33
 
 using Logging
-usempi = get(ENV, "CLIMACORE_DISTRIBUTED", "") == "MPI"
 if usempi
     using ClimaComms
     using ClimaCommsMPI
-    const context = ClimaCommsMPI.MPICommsContext()
-    const pid, nprocs = ClimaComms.init(context)
-
-    # log output only from root process
-    logger_stream = ClimaComms.iamroot(context) ? stderr : devnull
-
-    prev_logger = global_logger(ConsoleLogger(logger_stream, Logging.Info))
-    atexit() do
-        global_logger(prev_logger)
-    end
 else
     import TerminalLoggers
-    global_logger(TerminalLoggers.TerminalLogger())
-    context = nothing
 end
+using ClimaCorePlots
+import Plots
+
+# Physical parameters needed
+Base.@kwdef struct Params{FT}
+    R::FT = FT(6.37122e6)
+    Ω::FT = FT(7.292e-5)
+    g::FT = FT(9.80616)
```

Comment:
These are values we can get from CLIMAParameters

Reply:
We will do CLIMAParameters integration in a separate dedicated PR!